src/goose/console/models/redis/home.clj

```diff
@@ -0,0 +1,17 @@
+(ns goose.console.models.redis.home
```

Comment:
Why do you feel this deserves its own namespace?

Reply:
A page can consist of multiple components with their data.
A namespace (based on the page) unifies these different components. 
For example: Enqueued page consists of 2 different components. 
A sidebar and jobs-table. So the logic to construct the data model for sidebar and jobs-table would go `models.redis.enqueued-jobs `namespace.

Another alternative could be(i guess that what you are indirectly suggesting) defining the data-models and fetching it from the brokers all can go in single file i.e `goose.console.models.redis` . 
Since the logic would be small for all the pages, they all can belong to a single namespace. What do you think?